src/project.cc

```diff
@@ -290,12 +296,8 @@ std::vector<Project::Entry> LoadCompilationEntriesFromDirectory(
 
     our_time.Resume();
     std::string absolute_filename;
```

Comment:
Is `absolute_filename` still used? It doesn't seem to be initialized anymore. Remove it?